R/collect.R

```diff
@@ -269,6 +271,10 @@ prob_summarize <- function(x, p) {
   x
 }
 
+mean_na_rm <- function(x) {
```

Comment:
Do you think it's worth it leaving a comment here that this is to avoid using an anonymous function for speed reasons?

Reply:
Yes, good point.

Reply:
[658a1dc](https://github.com/tidymodels/tune/pull/655/commits/658a1dc5679792419f73c86460de2ad8be44e3ae)!